source/_UIAHandler.py

```diff
@@ -756,7 +756,9 @@ def _isUIAWindowHelper(self,hwnd):
 			canUseOlderInProcessApproach = bool(appModule.helperLocalBindingHandle)
 			isOfficeApp = appModule.productName.startswith(("Microsoft Office", "Microsoft Outlook"))

```

Comment:
Based on the log snipped I posted in #12852 I believe this fix would not be effective since `fileUtils` raises an error during execution of this line when trying to access `productName` of the `LogonUI`